### ISSUE TITLE
VS TestExplorer results don't include expected/actual if error message supplied

### ISSUE THREAD

**Issue by LukeAtLion** [mmanela/chutzpah#341]
I'm using the latest version of Chutzpah and VS 2013.  When I run tests in the Test Explorer window, if an assert fails and the assert includes a message, I see the message but not the expected and actual values.  If I remove the message from the assert, I see the expected and actual values but (obviously) no message.  I'd like to see both.

**Comment by mmanela** [comment]
Can you provide a basic repro?

**Comment by LukeAtLion** [comment]
My test file: Models.js

QUnit.module("Models");
QUnit.test("Test 2", function (assert)
{
    assert.equal(1, 2, "test 2");
});


My test file: ModelsTest.html


<!DOCTYPE html>

<html>

<head>

    <meta charset="utf-8">

    <title>Models Tests</title>

    

    <link rel="stylesheet" href="../../Utility/qunit-1.17.1.css">

</head>

<body>

    <div id="qunit"></div>

    <div id="qunit-fixture"></div>

    <script src="../../Utility/qunit-1.17.1.js"></script>



    

    <script src="ModelsTest.js"></script>

</body>

</html>

When Test 2 fails, I see:

[cid:image001.png@01D06555.063F9480]

The message I would prefer to see is:

Message: Expected: 2, Actual: 1, test 2

In other words, whether the user supplies a message or not, always display the Expected and Actual values when the test fails.

Thanks,

- Luke -

**Comment by LukeAtLion** [comment]
Not sure if the image can be seen. The image text is:
Test Failed - Models Test 2
Message: test 2

**Comment by mmanela** [comment]
I change the code to always append that actual and excepted.
